src/core/textures/Texture.js

```diff
@@ -140,6 +140,7 @@ export default class Texture extends EventEmitter
         {
             baseTexture.once('loaded', this.onBaseTextureLoaded, this);
         }
+        baseTexture.on('destroy', this.destroy, this);
```

Comment:
I realize that we still have problem if Texture is in TextureCache and BaseTexture was destroyed, so.. lets scan TextureCache periodically? :)